### ISSUE TITLE
Finite element field wrapper issues

### ISSUE THREAD

**Issue by rchristie** [cmlibs/zinc#151]
When finite element fields are created with legacy FE_field functions (including reading EX files) the cmzn_field_finite_element wrapper does not exist until after the last end change.
A further issue is that the finite element field wrapper is set to be managed even when created from public API (default should be unmanaged).

**Comment by rchristie** [comment]
Failing test; last isManaged() returns true:
```
TEST(ZincFieldStoredMeshLocation, managed)
{
	ZincTestSetupCpp zinc;

	Mesh mesh3d = zinc.fm.findMeshByDimension(3);
	zinc.fm.beginChange();
	FieldStoredMeshLocation fieldStoredMeshLocation = zinc.fm.createFieldStoredMeshLocation(mesh3d);
	EXPECT_FALSE(fieldStoredMeshLocation.isManaged());
	zinc.fm.endChange();
	EXPECT_FALSE(fieldStoredMeshLocation.isManaged());
}
```

**Comment by rchristie** [comment]
Additional issue: updating the computed field wrapper forces managed flag to be true.

**Comment by rchristie** [comment]
Closed by #179.
